lib/fedex_rest_api/track.rb

```diff
@@ -0,0 +1,39 @@
+require "fedex_rest_api/base"
+
+class FedexRestApi::Track
+  attr_reader :tracking_object
+
+  def initialize(tracking_object)
+    @tracking_object = tracking_object
+  end
+
+  def track
+    response = HTTParty.post(env_url,
+      headers: {
+        "Content-Type" => 'application/json',
+        "Authorization" => "Bearer #{tracking_object[:access_token]}"
```

Comment:
One question I have is, does it seem a bit cumbersome - and limiting - that this class needs to know that the "tracking object" is a hash with certain keys?  Any thought on passing in a struct or PORO that has methods `#access_token` and `#include_detailed_scans`, `#tracking_numbers`, and `#errors?`?  An approach like that might simplify and clarify the logic of when to raise errors for example and allow for further customization by overriding the class methods.

Reply:
I'm definitely not opposed to bolstering the interface! 

I also think the exact requirements for this are going to become self-evident as we start to build out logic from the mealhand side and weigh how much of this should live in a class/object, optional param object, initializer, etc...